src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,218 @@
+            car.setDrivers(getDriversByCarID(car.getId()));
+        }
+        return cars;
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "UPDATE cars "
+                + "SET model = ?, manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement
+                         = connection.prepareStatement(query)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.setLong(3, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't update "
+                    + car + " in carsDB.", e);
+        }
+        deleteDriverList(car);
+        createDriverList(car);
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete car with id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT c.id AS car_id, model, manufacturer_id, "
+                + "m.name as manufacturer_name, m.country "
+                + "FROM cars AS c "
+                + "INNER JOIN manufacturers AS m ON m.id = c.manufacturer_id "
+                + "INNER JOIN cars_drivers AS cd ON cd.car_id = c.id "
+                + "INNER JOIN drivers as d ON cd.driver_id = d.id "
+                + "WHERE cd.driver_id = ? AND c.is_deleted = FALSE AND d.is_deleted = FALSE;";
+        List<Car> cars = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(getCarFromResultSet(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get car's list by driver id"
+                    + driverId, e);
+        }
+        for (Car car: cars) {
+            car.setDrivers(getDriversByCarID(car.getId()));
+        }
+        return cars;
+    }
+
+    private void createDriverList(Car car) {
+        if (car.getDrivers().isEmpty()) {
+            return;
+        }
+        StringBuilder values = new StringBuilder();
+        for (var ignored : car.getDrivers()) {
```

Comment:
Code style should be consistent. You use particular type everywhere else but here you used var instead